tests/components/zha/test_api.py

```diff
@@ -244,21 +244,26 @@ async def test_list_groupable_devices(zha_client, device_groupable):
     assert msg["id"] == 10
     assert msg["type"] == const.TYPE_RESULT
 
-    devices = msg["result"]
-    assert len(devices) == 1
-
-    for device in devices:
-        assert device[ATTR_IEEE] == "01:2d:6f:00:0a:90:69:e8"
-        assert device[ATTR_MANUFACTURER] is not None
-        assert device[ATTR_MODEL] is not None
-        assert device[ATTR_NAME] is not None
-        assert device[ATTR_QUIRK_APPLIED] is not None
-        assert device["entities"] is not None
-
-        for entity_reference in device["entities"]:
+    device_endpoints = msg["result"]
+    assert len(device_endpoints) == 1
+
+    for endpoint in device_endpoints:
+        assert endpoint["device"][ATTR_IEEE] == "01:2d:6f:00:0a:90:69:e8"
+        assert endpoint["device"][ATTR_MANUFACTURER] is not None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)